.github/actions/generate-commit-list-v1/src/getRepositoryURL.ts

```diff
@@ -0,0 +1,24 @@
+import { getExecOutput } from '@actions/exec'
+
+/**
+ * Gets the URL of the current git repository.
+ * @returns A promise that resolves to a string containing the URL of the current git repository.
+ * @throws An error if the repository URL cannot be retrieved.
+ */
+
+export default async function getRepositoryURL(): Promise<string> {
+  try {
+    const { stdout: repositoryURL } = await getExecOutput('git', [
```

Comment:
Rather than doing this we should just require passing the repo in from the github action which already has this info available.